clang/test/Lexer/ms-extensions.c

```diff
@@ -13,16 +13,24 @@ __int64 w = 0x43ui64;
 __int64 z = 9Li64;  // expected-error {{invalid suffix}}
 __int64 q = 10lli64;  // expected-error {{invalid suffix}}
 
-__complex double c1 = 1i;
-__complex double c2 = 1.0i;
+__complex double c1 = 1i; // GNU extension
+__complex double c2 = 1.0i; // GNU extension
 __complex float c3 = 1.0if;
 
+#define UINT128_MAX 0xffffffffffffffffffffffffffffffffui128
```

Comment:
Maybe adding a test for `i128` suffix makes sense?

Reply:
Yeah, that is a good idea, I'll add that.